api/pacman-api-compliance/src/main/java/com/tmobile/pacman/api/compliance/repository/ComplianceRepositoryImpl.java

```diff
@@ -1723,12 +1723,13 @@ private static String getCurrentQuarterCriteriaKey() {
     @SuppressWarnings("rawtypes")
     public Map<String, Object> getPolicyViolationDetailsByIssueId(String assetGroup, String issueId)
             throws DataException {
-        Map<String, Object> issueDetails = new HashMap<>();
-        StringBuilder urlToQuery = new StringBuilder(esUrl).append("/").append(assetGroup);
-        urlToQuery.append("/").append(SEARCH).append("?").append("q").append("=").append("_id").append(":")
-                .append(issueId);
+    	Map<String, Object> issueDetails = new HashMap<>();
+		StringBuilder urlToQuery = new StringBuilder(esUrl).append("/").append(assetGroup);
+		urlToQuery.append("/").append(SEARCH);
 
-        StringBuilder requestBody = new StringBuilder();
+		StringBuilder requestBody = new StringBuilder();
+		requestBody.append("{\"query\":{\"bool\":{\"must\":[{\"match\":{\"type\":\"issue\"}},{\"match\":{\"_id\":\"")

```

Comment:
It would be ideal to use an ObjectMapper to create json content in a type safe and efficient manner